swagger.json

```diff
@@ -4362,6 +4362,37 @@
                 }
             ]
         },
+        "TaggedAddressBase": {
```

Comment:
Is `TaggedAddressBase` only for `TagCreate` purposes? (Asking because this is missing all of the other address related fields, like geofence, etc).

Reply:
It's actually only for the `GET` endpoint, single & list.